pkg/kudoctl/cmd/version.go

```diff
@@ -30,5 +34,40 @@ func newVersionCmd() *cobra.Command {
 func VersionCmd(cmd *cobra.Command, args []string) error {
 	kudoVersion := version.Get()
 	fmt.Printf("KUDO Version: %s\n", fmt.Sprintf("%#v", kudoVersion))
+
+	// Print the Controller Version
+	controllerVersion, err := GetControllerVersion()
+	if err != nil {
+		fmt.Printf("KUDO Controller Version: %s\n", controllerVersion)
+	} else {
+		fmt.Printf("KUDO Controller Version: %#v\n", err)
+	}
+
 	return nil
 }
+
+// GetControllerVersion
+func GetControllerVersion() (string, error) {
+
+	controllerVersion := ""
+
+	client, err := kube.GetKubeClient(Settings.KubeConfig)
+	clog.V(3).Printf("Acquiring kudo client")
+	if err != nil {
+		clog.V(3).Printf("Failed to acquire kudo client")
+		return "", errors.New("<Failed to acquire kudo client>")
+	}
+
+	statefulsets, err := client.KubeClient.AppsV1().StatefulSets("").List(metav1.ListOptions{LabelSelector: "app=kudo-manager"})
+	clog.V(3).Printf("List statefulsets and filter kudo-manager")
+	if err != nil {
+		clog.V(3).Printf("Failed to list kudo-manager statefulset")
+		return "", errors.New("<Error: Failed to list kudo-manager statefulset>")
+	}
+
+	for _, d := range statefulsets.Items {
+		controllerVersion = d.Spec.Template.Spec.Containers[0].Image
+	}
```

Comment:
There's a lot of assumptions here. Let's be a bit more strict about them. We assume that the `StatefulSet` runs a single container which is the manager. Let's add checks to double down on these assumptions:
```
if len(statefulsets.Items) > 1 {
  return errors.New("more than 1 KUDO controller manager running")
}
if len(statefulsets.Items[0].Spec.Template.Spec.Containers) != 1 {
  return fmt.Errorf("invalid number of containers in statefulset %s/%s", statefulsets.Items[0].Namespace, statefulsets.Items[0].Name)
}
controllerVersion := statefulsets.Items[0].Spec.Containers[0].Image
```

Reply:
Oh, and we also have to handle the case that KUDO isn't deployed on the cluster yet.